tensorflow/lite/micro/python/interpreter/tests/test_model.py

```diff
@@ -0,0 +1,296 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Create a sample model for testing purposes.
+
+A simple quantized int8 model with single ADD operation.
+"""
+
+import flatbuffers
+import numpy as np
+
+from tflite_micro.tensorflow.lite.python import schema_py_generated as schema_fb
+
+TFLITE_SCHEMA_VERSION = 3
+
+
+def _generate_quantization_parameters(builder, rmin, rmax, qmin, qmax):
+  """Tensor-wise quantization parameter generation
```

Comment:
It's not too hard to guess in this case, but I think some comments on what the input arguments are would be worthwhile.